app/pages/admin/featured-project-toggle.spec.js

```diff
@@ -8,12 +8,13 @@ let checkbox;
 let featuredChangeSpy;
 let wrapper;
 const project = {
-  featured: false
+  featured: false,
+  launch_approved: true
 };
 
 function setup() {
   featuredChangeSpy = sinon.spy();
-  wrapper = shallow(
+  wrapper = mount(
```

Comment:
`FeaturedProjectToggle` is a stateless function, with no children, so why is it using `mount`?

Reply:
My bad, sorry